pkg/hubble/parser/parser.go

```diff
@@ -26,6 +28,8 @@ import (
 	monitorAPI "github.com/cilium/cilium/pkg/monitor/api"
 	"github.com/cilium/cilium/pkg/proxy/accesslog"
 	"github.com/golang/protobuf/ptypes"
+	"google.golang.org/protobuf/types/known/timestamppb"
+	"google.golang.org/protobuf/types/known/wrapperspb"
```

Comment:
Most Hubble code predates the v2 `google.golang.org/protobuf` package (c.f. https://blog.golang.org/protobuf-apiv2) and uses v1 imports instead:
```
"github.com/golang/protobuf/ptypes/timestamp"
"github.com/golang/protobuf/ptypes/wrappers"
```

Both import paths import the same types (the v1 imports are aliases for the v2 imports). We should ideally update the code base to use the new paths everywhere, but that's unrelated to the PR. 

What is important to me however is that we are consistent within a single package, so we should either remove the import of `"github.com/golang/protobuf/ptypes/wrappers"` below (see comment) or import `"github.com/golang/protobuf/ptypes/timestamp"` here.

Reply:
Thanks for the explanation. Being a protobuf newbie I didn't pay much attention to this and just used the packages `goimports` added for me :see_no_evil:

Given that `github.com/golang/protobuf/ptypes/wrappers` is already imported, I'll use the v1 imports for this PR in order to stay consistent. I could send a follow-up PR to update the entire code base if you think that's something worth doing.